script/glam/generate_sql

```diff
@@ -12,7 +12,7 @@ tables=$(bq ls --format=json $project:$dataset | \
 
 function write_scalars {
     local table=$1
-    local directory="sql/glam_etl/fenix_clients_daily_scalar_aggregates_${table}"
+    local directory="sql/glam_etl/${dataset/_stable/}__clients_daily_scalar_aggregates_${table}"
```

Comment:
`${dataset/_stable/}` is used as much as `$dataset` so it might be worth using a, `app_id` variable like `${app_id}_stable` instead of `dataset`.  I think this would be clearer on the intention of the variable by turning app id into dataset instead of the other way around.

edit: or `product` instead of `app_id` to match `run_glam_sql`